src/js/index.js

```diff
@@ -1,17 +1,175 @@
-const $showResultButton = document.querySelector('.open-result-modal-button')
-const $modalClose = document.querySelector('.modal-close')
-const $modal = document.querySelector('.modal')
-const $lottoNumbersToggleButton = document.querySelector(
-  '.lotto-numbers-toggle-button'
-)
-
-const onModalShow = () => {
-  $modal.classList.add('open')
-}
+const $ = (selector) => document.querySelector(selector);
+
+function App() {
+  let lottoTicketsList = [];
+  let IsLottoNumberVisible = false;
+
+  const isValidPrice = (price) => {
+    if (price === "") {
+      alert("금액을 입력해주세요");
+      return false;
+    }
+    if (price < 1000 || 100000 < price) {
+      alert("천원 이상 십 만원 이하로 입력해주세요.");
+      return false;
+    }
+    if (price % 1000 !== 0) {
+      alert("금액을 천원 단위로 입력해주세요");
+      return false;
+    }
+    return true;
+  };
+
+  const generateRandomNumber = (min, max) => {
+    return Math.floor(Math.random() * (max - min) + min);
+  };
+
+  const showLottoNumbers = () => {
+    $(".lotto-tickets")
+      .querySelectorAll(".lotto-tickets-numbers")
+      .forEach((el) => {
+        el.classList.remove("d-none");
+        el.classList.add("d-flex");
+      });
+    IsLottoNumberVisible = false;
+  };
+
+  const hideLottoNumbers = () => {
+    $(".lotto-tickets")
+      .querySelectorAll(".lotto-tickets-numbers")
+      .forEach((el) => {
+        el.classList.remove("d-flex");
+        el.classList.add("d-none");
+      });
+    IsLottoNumberVisible = true;
+  };
+
+  const updateLottoAmounts = (amounts) => {
+    $(".lotto__menu").innerHTML = `
+      <label class="flex-auto my-0 lotto-amounts"
+      >총 ${amounts}개를 구매하였습니다.</label
+      >
+      <div class="flex-auto d-flex justify-end pr-1">
+        <label class="switch">
+          <input type="checkbox" class="lotto-numbers-toggle-btn" />
+          <span class="text-base font-normal">번호보기</span>
+        </label>
+      </div>
+    `;
+  };
+
+  const generateLottoNumbers = (amounts) => {
+    for (let i = 0; i < amounts; i++) {
```

Comment:
map을 통해서 만들면 더 가독성이 좋을 것 같아요 
```suggestion
    new Array(amounts).map(_ =>generateRandomNumber(1, 99) )
```

Reply:
이런 좋은 방법이 있었네요!! 앞으로 반복문으로 배열 만드는 것 대신 써보겠습니다!!